teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<ul><li>Sometimes, ADME(T) also includes&nbsp;<strong>T</strong>oxicology. -&gt; Often, ADME also includes&nbsp;<strong>T</strong>oxicology and is thus referred to as ADMET or ADMETox.</li><li>The amount and the time drug-uptake ... -&gt; The amount and the time <u>of</u> drug-uptake ...</li><li>The distribution of an absorbed substance, i.e. within the body, between blood and different tissues, and crossing of the blood-brain barrier are affected by regional blood flow rates, molecular size and polarity of the compound, and binding to serum proteins and transporter enzymes. -&gt; The distribution of an absorbed substance, i.e. within the body, between blood and different tissues, <u>or</u> crossing <s>of</s> the blood-brain barrier are affected by regional blood flow rates, molecular size and polarity of the compound, <u>as well as</u> binding to serum proteins and transporter enzymes.</li><li>As soon as a compound enters the body, it usually starts to be metabolized. -&gt; Maybe: After entering the body, the compound will be metabolized.</li><li>Sentence? This way, toxic compounds can be removed from the body, however the metabolism itself can also Reducing the amount of an absorbed substance can be favorable if a toxic compound is removed. transformation of a chemical could even yield new toxic metabolites. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Thanks. Regarding the last point, I have deleted the comment on toxic molecules, since the focus is here on the metabolism.</p>